Doc/library/graphlib.rst

```diff
@@ -19,20 +19,20 @@
 
    Provides functionality to topologically sort a graph of hashable nodes.
 
-   A topological order is a linear ordering of the vertices in a graph such that
-   for every directed edge u -> v from vertex u to vertex v, vertex u comes
-   before vertex v in the ordering. For instance, the vertices of the graph may
+   A topological order is a linear ordering of the nodes in a graph such that
+   for every directed edge u -> v from node u to node v, node u comes
+   before node v in the ordering. For instance, the nodes of the graph may
    represent tasks to be performed, and the edges may represent constraints that
    one task must be performed before another; in this example, a topological
    ordering is just a valid sequence for the tasks. A complete topological
-   ordering is possible if and only if the graph has no directed cycles, that
-   is, if it is a directed acyclic graph.
+   ordering is possible if and only if the graph is a directed acyclic graph,
+   that is, if there are no circular dependencies among the tasks to be performed.
```

Comment:
I feel that `tasks to be performed`  is one specific application of graphs, but not a generic term enough to fit in this doc.

Reply:
If only that were true...
Alas it is clearly stated that the primary purpose of this class is to do task dependency resolution.

> The class is designed to easily support parallel processing of the nodes as they become ready.